inc/Engine/Admin/HealthCheck.php

```diff
@@ -117,4 +96,36 @@ public function missed_cron() {
 			]
 		);
 	}
+
+	/**
+	 * Checks if health check should not run.
+	 *
+	 * @since 3.5.4
+	 *
+	 * @return bool true when should not do health check; else, false.
+	 */
+	protected function do_not_check() {
```

Comment:
@hellofromtonya what do you think of using a name like `should_check` instead? And of course inverting the returns inside the method.

I tend to prefer avoiding names with a negation, as it makes it more difficult to read for me when you have to use it with `!` for example

Reply:
@Tabrisrp That's fine. You'll need to invert the logic in the method.